html/id.html

```diff
@@ -27,7 +27,7 @@ <h2 class="colors">Purple</h2>
     <h2 class="colors">Black</h2>
     <h2 class="colors">Pink</h2>
     <h2 id="C4">Red</h2>
```

Comment:
Suggestion:
- the id or class should have a specific format, the value of it should have dash between word and lowercap letters. e.q. my-id or car-part-panel.